framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -1243,8 +1243,7 @@ public void computeVarargType() {
          *
          * @param annotatedExecutableType an AnnotatedExecutableType
          */
-        /*package-private*/ void computeVarargType(
-                AnnotatedExecutableType annotatedExecutableType) {
+        public void computeVarargType(AnnotatedExecutableType annotatedExecutableType) {
```

Comment:
There is a public `computeVarargType` above.
Could that method be used instead? If not, can you add javadoc to make the relation clear?